pwnlib/commandline/hex.py

```diff
@@ -16,14 +16,40 @@
 parser.add_argument('data', nargs='*',
     help='Data to convert into hex')
 
+parser.add_argument(
+    '-p', '--prefix',
+    metavar = 'prefix',
+    type = str,
+    default = '',
+    help = 'Insert a prefix before each byte',
+)
+
+parser.add_argument(
+    '-s', '--separator',
+    metavar = 'separator',
+    type = str,
+    default = '',
+    help = 'Add a separator between each byte',
+)
+
+def format_hex(s, prefix, separator):
+    n = 2
+    parts = [s[i:i+n] for i in range(0, len(s), n)]
```

Comment:
The same thing is done by [`util.iters.group`](https://docs.pwntools.com/en/latest/util/iters.html#pwnlib.util.iters.group) or [`util.lists.group`](https://docs.pwntools.com/en/latest/util/lists.html#pwnlib.util.lists.group) already.